ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -96,21 +101,53 @@ platform:
     externalMACAddress: $(<"${SHARED_DIR}/ipi_bootstrap_mac_address")
     hosts: []
 "
+  yq --inplace eval-all 'select(fileIndex == 0).platform.baremetal = select(fileIndex == 1) | select(fileIndex == 0)' \
+    "$SHARED_DIR/install-config.yaml" - <<< "$ADAPTED_YAML"
+else
+  ADAPTED_YAML="
+    libvirtURI: >-
+      qemu+ssh://root@${AUX_HOST}:$(<"${CLUSTER_PROFILE_DIR}/provisioning-host-ssh-port-${architecture}")/system?keyfile=${CLUSTER_PROFILE_DIR}/ssh-key&no_verify=1&no_tty=1
+    apiVIP: $(yq ".api_vip" "${SHARED_DIR}/vips.yaml")
+    ingressVIP: $(yq ".ingress_vip" "${SHARED_DIR}/vips.yaml")
+    externalMACAddress: $(<"${SHARED_DIR}/ipi_bootstrap_mac_address")
+    provisioningNetwork: Disabled
```

Comment:
could we leverage two additional steps and the patching system we have in the other PR?

We'd have a step before the install one to provide the patch for this snipped (and the hosts one too probably) in the case of provisioning network and another step for the case of redfish-based workflows.